src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -0,0 +1,134 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ===================================
+# Schema for:
+#  * 'show wireless mobility summary'
+# ===================================
+class ShowWirelessMobilitySummarySchema(MetaParser):
+    """Schema for show wireless mobility summary."""
+
+    schema = {
+        "controller_config": {
+            "group_name": str,
+            "ipv4": str,
+            "mac_address": str,
+            "multicast_ipv4": str,
+            "multicast_ipv6": str,
+            "pmtu": str,
+            "public_ip": str,
+            "status": str,
+        },
+        "mobility_summary": {
+            "domain_id": str,
+            "dscp_value": str,
+            "group_name": str,
+            "keepalive": str,
+            "mac_addr": str,
+            "mgmt_ipv4": str,
+            "mgmt_ipv6": str,
+            "mgmt_vlan": str,
+            "multi_ipv4": str,
+            "multi_ipv6": str,
+        },
+    }
+
+
+# ===================================
+# Parser for:
+#  * 'show wireless mobility summary'
+# ===================================
+class ShowWirelessMobilitySummary(ShowWirelessMobilitySummarySchema):
+    """Parser for show wireless mobility summary"""
+
+    cli_command = ["show wireless mobility summary"]
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command[0])
+
+        else:
+            output = output
+
+        # Mobility Summary
+
+        # Wireless Management VLAN: 299
+        # Wireless Management IP Address: 10.10.7.177
+        # Wireless Management IPv6 Address:
+        # Mobility Control Message DSCP Value: 48
+        # Mobility Keepalive Interval/Count: 10/3
+        # Mobility Group Name: b80-mobility
+        # Mobility Multicast Ipv4 address: 0.0.0.0
+        # Mobility Multicast Ipv6 address: ::
+        # Mobility MAC Address: 58bf.ea35.b60b
+        # Mobility Domain Identifier: 0x61b3
+
+        # Controllers configured in the Mobility Domain:
+
+        # IP                                        Public Ip                                  MAC Address         Group Name                       Multicast IPv4    Multicast IPv6                              Status                       PMTU
+        # --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+        # 10.10.7.177                               N/A                                        58bf.ea35.b60b      b80-mobility                0.0.0.0           ::                                          N/A                          N/A
+
+        # Mobility Summary
+        mobility_summary_capture = (
```

Comment:
Do you feel this is easily readable?

Reply:
No I do not, I thought we were supposed to comment the raw output for reference. I can reformat it with just single lines

Reply:
I do! PR submitted!